### PR TITLE
Update replicationcontroller.md

### PR DESCRIPTION
**Pull request by my-git9**
Sync English documentation about replicationcontroller

### CODE REVIEW

**Comment by tengqm** [comment]
/hold

By running the following command on the **main** branch, you could find all the changes that need to be synced:

```
./scripts/lsync.sh content/zh/docs/concepts/workloads/controllers/replicationcontroller.md
```

**Comment by my-git9** [comment]
> Modifications have been made based on your suggestion, thanks for you review

File: content/zh/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 1]
```diff
@@ -596,11 +594,19 @@ ReplicationController。
...
+  Read the {{< api-reference page="workload-resources/replication-controller-v1" >}}
+  object definition to understand the API for replication controllers.
 -->
-## 更多信息
+## 接下来
 
-请阅读[运行无状态的 ReplicationController](/zh/docs/tasks/run-application/run-stateless-application-deployment/)。
+- 了解关于Pods[/zh/docs/concepts/workloads/pods]。
+- 了解[Depolyment](/zh/docs/concepts/workloads/controllers/deployment/)，ReplicationController的替代品。
+- `ReplicationController`是 Kubernetes REST API 的一部分，阅读 {{< api-reference page="workload-resources/replication-controller-v1" >}}
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/replicationcontroller.md:610]:
```suggestion
- `ReplicationController` 是 Kubernetes REST API 的一部分，阅读
  {{< api-reference page="workload-resources/replication-controller-v1" >}}
```

**Pull request merged**